cli/delete.go

```diff
@@ -83,3 +97,24 @@ func deleteRecordCommand(t *core.Timetrace) *cobra.Command {
 
 	return deleteRecord
 }
+
+func askForConfirmation() bool {
+	reader := bufio.NewReader(os.Stdin)
+	for {
+		fmt.Fprint(os.Stderr, "Please confirm (Y/N): ")
+		s, _ := reader.ReadString('\n')
+		s = strings.TrimSuffix(s, "\n")
+		s = strings.ToLower(s)
+		if len(s) > 1 {
+			continue
+		}
+		if strings.Compare(s, "n") == 0 {
+			return false
+		} else if strings.Compare(s, "y") == 0 {
+			break
+		} else {
+			continue
+		}
```

Comment:
```suggestion
		}
```

This can be removed 😄

Reply:
😊You are right.